dll/win32/shell32/lang/pl-PL.rc

```diff
@@ -46,7 +46,7 @@ BEGIN
         MENUITEM "&Odśwież", FCIDM_SHVIEW_REFRESH
         MENUITEM "", -1, MFT_SEPARATOR
         MENUITEM "W&klej", FCIDM_SHVIEW_INSERT
-        MENUITEM "Wklej s&krót", FCIDM_SHVIEW_INSERTLINK
+        MENUITEM "Wklej &skrót", FCIDM_SHVIEW_INSERTLINK
```

Comment:
yes, this fixes an accelerator collision. And yes: this is better than it was. But still you base here (and in many other places) your changes solely on guessing. You do have a 50% chance that you guessed right, but there is also a 50% chance that you guessed wrong, and that it would actually be "W&klej" that would need to be changed (or that &s is still the wrong accel). I am not sure, whether that is a clever way to approach things.

Reply:
Ok, updated the preceeding item. Not guessing here, just selecting an available key that is not in use.  (that's what I usually do when I can't confirm the key explicitly by looking at Windows)

Reply:
Yes, this is the definition of guessing. There is a chance, that you make it worse, if guessing wrong. We therefore should give the reviewers a bit of time at least to intercept where they see the necessity.

Reply:
Sure, no rush here.